src/direct.js

```diff
@@ -1,16 +1,16 @@
-const utils = require('./common/utils'),
-  constants = require('./common/constants'),
-  Queue = utils.Queue,
-  WavefrontSdkMetricsRegistry = require('./common/metrics/registry')
-    .WavefrontSdkMetricsRegistry,
-  https = require('https'),
-  zlib = require('zlib');
+import { Queue } from './common/utils';
+import * as utils from './common/utils';
+import * as constants from './common/constants';
+import WavefrontSDKMetricsRegistry from './common/metrics/registry';
+
+require('isomorphic-fetch');
+const pako = require('pako');
```

Comment:
Can these be `import`ed?

Reply:
yah I'm struggling about the inconsistency as well. I tried to `import` the libraries but there seems to be some error when it compiles to UMD bundles. The libraries are not compiled to ES5 syntax.

Reply:
My guess is you need another plugin in your rollup config: https://github.com/rollup/plugins/tree/master/packages/commonjs

Reply:
commonjs will probably also require dynamicRequireTargets

Reply:
@aliu-vmware I've tried to compile with `commonjs` plugin but it seems that `isomorphic-fetch` still cannot compile properly, thus resulting in errors when using the bundle from tier 2 SDK. I'm wondering if we can leave the `require` statement as is for now and probably go back to revise later?

Reply:
Since this is a non-trivial fix, I created another story to keep track of this later. We'll require the module as is for now.